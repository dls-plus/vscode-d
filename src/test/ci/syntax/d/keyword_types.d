void main() {
	Joint j;
	cast(Joint)4;
	debugv = 5;
}