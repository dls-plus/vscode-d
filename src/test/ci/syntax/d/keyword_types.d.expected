[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":9,"scope":"entity.name.function.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":14,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":6,"scope":"variable.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":8,"scope":"variable.d"},{"start":8,"end":10,"scope":"meta.statement.end.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":6,"scope":"support.type.cast.begin.d"},{"start":6,"end":11,"scope":"source.d"},{"start":11,"end":12,"scope":"support.type.cast.end.d"},{"start":12,"end":13,"scope":"constant.numeric.integer.decimal.d"},{"start":13,"end":15,"scope":"meta.statement.end.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":7,"scope":"variable.d"},{"start":7,"end":8,"scope":"source.d"},{"start":8,"end":9,"scope":"keyword.operator.assign.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"constant.numeric.integer.decimal.d"},{"start":11,"end":13,"scope":"meta.statement.end.d"}]
[{"start":0,"end":2,"scope":"source.d"}]